packs/data/blood-lords-bestiary.db/drusilla.json

```diff
@@ -1311,7 +1311,7 @@
                     "value": 1
                 },
                 "description": {
-                    "value": "<p>Drusilla touches another creature to manipulate the creature's luck. The creature must attempt a @Check[type:will|dc:25] save. On a failure, Drusilla chooses good luck or bad luck. If she chooses good luck, the affected creature can roll twice on one d20 roll within the next day and use the higher result; this is a fortune effect. If Drusilla chooses bad luck, the creature must roll twice and use the lower result on its next d20 roll; this is a misfortune effect. Once Drusilla has affected a target with Manipulate Luck, that target is temporarily immune to Manipulate Luck from any huldra for 1 day.</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Effect: Manipulate Luck (Good)]{Effect: Manipulate Luck (Good)}</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Effect: Manipulate Luck (Bad)]{Effect: Manipulate Luck (Bad)}</p>"
+                    "value": "<p>Drusilla touches another creature to manipulate the creature's luck. The creature must attempt a @Check[type:will|dc:25] save. On a failure, Drusilla chooses good luck or bad luck. If she chooses good luck, the affected creature can roll twice on one d20 roll within the next day and use the higher result; this is a fortune effect. If Drusilla chooses bad luck, the creature must roll twice and use the lower result on its next d20 roll; this is a misfortune effect. Once Drusilla has affected a target with Manipulate Luck, that target is temporarily immune to Manipulate Luck from any huldra for 1 day.</p>\n<p>@UUID[Compendium.pf2e.campaign-effects.Effect: Manipulate Luck - Drusilla (Good)]{Effect: Manipulate Luck - Drusilla (Good)}</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Effect: Manipulate Luck (Bad)]{Effect: Manipulate Luck (Bad)}</p>"
```

Comment:
Can the duration be predicated?

Reply:
Ah, damn technically unlimited duration on the bad luck. I doubt it would practically matter in any games but we can keep it as two effects. One of these days we should figure out duration overriding.